src/content/ja/2019/caching.md

```diff
@@ -0,0 +1,691 @@
+  </a>
+  <div id="fig16-description" class="visually-hidden">応答の97％を示す棒グラフはSet-Cookieを使用せず、3％が使用します。この3％の内、15.3％がプライベート、84.7％がデスクトップ、モバイルは18.4％がパブリック、81.6％がプライベートであるという別の棒グラフに拡大されています。</div>
+  <figcaption id="fig16-caption">図16. <code>Set-Cookie</code>応答のキャッシュ可能な応答。</figcaption>
+</figure>
+
+## AppCacheおよびサービスワーカー
+
+アプリケーションキャッシュまたはAppCacheはHTML5の機能であり、開発者はブラウザがキャッシュするリソースを指定し、オフラインユーザーが利用できるようにできます。この機能は[廃止、Web標準から削除](https://html.spec.whatwg.org/multipage/offline.html#offline)され、ブラウザーのサポートは減少しています。実際、その使用が検出された場合、[Firefox v44 +は、開発者が代わりにサービスワーカーを使用することを推奨しています](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API/Using_Service_Workers)。 [Chrome 70は、アプリケーションキャッシュをセキュリティで保護されたコンテキストのみに制限します](https://www.chromestatus.com/feature/5714236168732672)。業界は、このタイプの機能をサービスワーカーに実装する方向へ移行しており、[ブラウザサポート](https://caniuse.com/#feat=serviceworkers)は急速に成長しています。
+
+実際、[HTTPアーカイブトレンドレポートの1つは、以下に示すサービスワーカー](https://httparchive.org/reports/progressive-web-apps#swControlledPages)の採用を示しています。
+
+<figure>
+  <a href="/static/images/2019/16_Caching/ch16_fig14_service_worker_adoption.jpg">
+    <img src="/static/images/2019/16_Caching/ch16_fig14_service_worker_adoption.jpg" alt="図17.サービスワーカーが制御するページの時系列。" aria-labelledby="fig17-caption" aria-describedby="fig17-description" width="600" height="311">
+  </a>
+  <div id="fig17-description" class="visually-hidden">2016年10月から2019年7月までのサービスワーカーが制御するサイトの使用状況を示す時系列チャート。モバイルとデスクトップの両方で使用量は年々着実に増加していますが、依然として両方で0.6％未満です。</div>
+  <figcaption id="fig17-caption">図17.サービスワーカーが制御するページの時系列。 (引用: <a href="https://httparchive.org/reports/progressive-web-apps#swControlledPages">HTTPアーカイブ</a>)</figcaption>
+</figure>
+
+採用率はまだウェブサイトの1％を下回っていますが、2017年1月から着実に増加しています。[プログレッシブWebアプリ](./pwa)の章では、人気サイトでの使用によりこのグラフが示唆するよりも多く使用されているという事実を含め、上記のグラフでは1回のみカウントされます。
+
+次の表では、AppCacheとService Workerの使用状況の概要を確認できます。 32,292のWebサイトでサービスワーカーが実装されていますが、1,867のサイトでは非推奨のAppCache機能が引き続き使用されています。
+
+<figure>
+  <table>
+    <tr>
+     <td></td>
+     <th>サーバーワーカーを使用しない</th>
+     <th>サーバーワーカーを使用する</th>
+     <th>合計</th>
+    </tr>
+    <tr>
+     <td>AppCacheを使用しない</td>
+     <td><p style="text-align: right">5,045,337</p></td>
+     <td><p style="text-align: right">32,241</p></td>
+     <td><p style="text-align: right">5,077,578</p></td>
+    </tr>
+    <tr>
+     <td>AppCacheを使用する</td>
+     <td><p style="text-align: right">1,816</p></td>
+     <td><p style="text-align: right">51</p></td>
+     <td><p style="text-align: right">1,867</p></td>
+    </tr>
+    <tr>
+     <td>合計</td>
+     <td><p style="text-align: right">5,047,153</p></td>
+     <td><p style="text-align: right">32,292</p></td>
+     <td><p style="text-align: right">5,079,445</p></td>
+    </tr>
+  </table>
+  <figcaption>図18. AppCacheを使用するWebサイトとサービスワーカーの数。</figcaption>
```

Comment:
Depending on whether you agree with me above you may want to bring back "service workers" here too. And in fact all instances of service workers.

Reply:
"service workers" is is not HTTP Header, So I don't thinsk that this is translate.

Reply:
This line fix too.